packages/prerender/@babel/register.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@babel/register' {
+  export default function (options: unknown): void
+}
```

Comment:
is it possible to stick this in ambient.d.ts?

Reply:
There was a reason I didn't do this, can't remember exactly. Either a typescript/eslint error or that it wasn't picking up the type definition. I'll give it a shot again, and get back to you

Reply:
![Screenshot 2021-01-27 at 14 13 23](https://user-images.githubusercontent.com/1521877/106003148-d25d4700-60a9-11eb-96e6-45d77d8151e8.png)
Yep, suggested by TS.